common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2229,6 +2229,11 @@
           "Partition key <partKey> must set value."
         ]
       },
+      "FUNCTION_WITH_UNSUPPORTED_SYNTAX" : {
+        "message" : [
+          "Function <prettyName> does not support <syntax>."
+        ]
+      },
```

Comment:
@MaxGekk I putted the `FUNCTION_WITH_UNSUPPORTED_SYNTAX` below `INVALID_SQL_SYNTAX`.
Is it suitable?